src/main/scala/gnieh/diffson/pointer.scala

```diff
@@ -61,14 +61,10 @@ object / {
 }
 
 private case object Root extends Pointer {
-  override def toString =
-    "/"
+  override def toString = ""
```

Comment:
Sorry about that, fixed the tests. Per https://tools.ietf.org/html/rfc6901, "" represents the root (the whole document) and "/" represents the value with key "", one level down from the root. (E.g. "//" refers to "foo" in the document { "": { "": "foo" } } )